app/src/main/java/com/example/seoulgonggong/ui/facility/SportsFacilityViewModel.kt

```diff
@@ -12,11 +17,53 @@ class SportsFacilityViewModel @Inject constructor(
     private val repository: SportsFacilityRepository,
 ) : ViewModel() {
 
+    private val _sportsFacilities: MutableLiveData<List<UiSportsFacility>> = MutableLiveData()
+    val sportsFacilities: LiveData<List<UiSportsFacility>> = _sportsFacilities
+
+    val searchWord: MutableLiveData<String> = MutableLiveData("")
+    private val _listSportsFacilities: MutableLiveData<UiSportsFacilityList> = MutableLiveData()
+    val listSportsFacilities: LiveData<UiSportsFacilityList> = _listSportsFacilities
+
+    private val _listOpenEvent: MutableLiveData<Boolean> = MutableLiveData()
+    val listOpenEvent: LiveData<Boolean> = _listOpenEvent
+
+    private val _detailOpenEvent: MutableLiveData<UiSportsFacility> = MutableLiveData()
+    val detailOpenEvent: LiveData<UiSportsFacility> = _detailOpenEvent
+
     fun getData() {
         viewModelScope.launch {
-            repository.getSportsFacility().onSuccess {
-                // 데이터 활용하기
+            repository.getSportsFacility().onSuccess { facilities ->
+                val uiFacilities = mutableListOf<UiSportsFacility>()
+                facilities.forEach {
+                    if (!it.info.type.contains(EXCLUDE_WORD)) uiFacilities.add(it.toUi())
+                }
+                _sportsFacilities.value = uiFacilities.toList()
```

Comment:
```suggestion
                val uiFacilities = facilities.filter {
                        !it.info.type.contains(EXCLUDE_WORD)
                    }.map {
                        it.toUi()
                    }
                    _sportsFacilities.value = uiFacilities
```
이렇게 바꿔서 mutableList로 선언 후 추가하는게 아니라 아예 처음부터 List로 만들어줄 수 있을 것 같아요!